README.md

```diff
@@ -249,6 +249,16 @@ It is due to invalid archs, search for them and check it. Maybe you
 targetted a simulator but have only armv7/arm64. Maybe you want to target
 your iPad but it as only x86_64.
 
+### Why does the python multiprocess/subprocess module not work?
+
+The iOS application model does not currently support multi-processing in a
+cross-platform compatible way. The application design focuses on minizing
```

Comment:
Just a minor typo: "minizing", then LGTM. (Maybe a `squash` before merging it)

Reply:
Done. Easiest to squash when you merge. There is that option is the github merge options? Thanks. Great to be thorough :+1: